packages/models/change_model/lib/src/change.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:meta/meta.dart';
+
+part 'change.g.dart';
+
+@JsonSerializable()
+class ChangeModel {
```

Comment:
Subjective: the naming is not so simple to understand. From the first sight, its impossible to tell what a *Change* do. I would propose *impact*. I dont know if you planning to make another model for suggestions, but please keep in mind that we are going to have suggestions (hints) as well.

Reply:
Must I change name of this model now?

Reply:
@bgoncharuck depending if my argument sounds reasonable. I'm not strong on that.